checker/build.xml

```diff
@@ -245,7 +245,7 @@
         <echo message="${qual.src.files.spaceseparated}" file="${tmpdir}/srcfiles-checker.txt"/>
         <java fork="true"
               failonerror="true"
-              classpath="${javac.lib}:${framework.lib}:${junit.lib}"
+              classpath="${javac.lib}:${framework.lib}:${junit.lib}:${afu.lib}"
```

Comment:
The changes that use the AFU JAR are in checker.jar, so why is adding it to the classpath needed when compiling checker-qual.jar and checker-compat-qual.jar ?